mergin/client.py

```diff
@@ -434,23 +435,47 @@ def create_project(self, project_name, is_public=False, namespace=None):
         Create new project repository in user namespace on Mergin Maps server.
         Optionally initialized from given local directory.
 
-        :param project_name: Project name
+        :param project_name: Project's full name (<namespace>/<name>)
         :type project_name: String
 
         :param is_public: Flag for public/private project, defaults to False
         :type is_public: Boolean
 
-        :param namespace: Optional namespace for a new project. If empty username is used.
+        :param namespace: Deprecated. project_name should be full project name. Optional namespace for a new project. If empty username is used.
         :type namespace: String
         """
         if not self._user_info:
             raise Exception("Authentication required")
 
+        if namespace and "/" not in project_name:
+            warnings.warn(
+                "The usage of `namespace` parameter in `create_project()` is deprecated."
+                "Specify `project_name` as full name (<namespace>/<name>)) instead.",
+                category=DeprecationWarning,
+            )
+
+        if "/" in project_name:
+            if namespace:
+                warnings.warn(
+                    "Parameter `namespace` specified with full project name (<namespace>/<name>)."
+                    "The parameter will be ignored."
+                )
+
+            splitted = project_name.split("/")
```

Comment:
splitted: https://english.stackexchange.com/questions/467873/the-past-participle-of-split-split-or-splitted#467898

I would suggest to just use:
```python
namespace, project_name = project_name.split("/")
```